core/test/oracle/Voting.js

```diff
@@ -119,6 +123,29 @@ contract("Voting", function(accounts) {
     assert.equal((await voting.getVotePhase()).toString(), VotePhasesEnum.COMMIT);
   });
 
+  it("Should snapshot only with valid EOA", async function() {
+    // Reset the rounds.
+    await moveToNextRound(voting);
+    await moveToNextPhase(voting);
+
+    // no sig passed should fail
```

Comment:
Small nit: our comments normally start with a capital letter and end with a full stop.